src/commands/server/news/edit.ts

```diff
@@ -42,21 +34,22 @@ export default class Edit {
     const user = interaction.user;
     const guild = interaction.guild;
     const id = interaction.options.getString("id", true).trim();
-    const news = await newsTable?.get(`${guild.id}.news.${id}`).then(
-      (news) => news as News
-    ).catch(() => null as News | null);
+    const news = await newsTable
+      ?.get(`${guild.id}.news.${id}`)
```

Comment:
```suggestion
    const news = await newsTable?
      .get(`${guild.id}.news.${id}`)
```
That character belongs there I think ;)